Pascal's_Triangle/Pascal's_Triangle.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+
+int factorial(int n)
+{
+   int f;
+   for(f = 1; n > 1; n--)
+      f *= n;
+   return f;
+}
+
+int main() 
+{
+   int n, i, j;
+   printf("Enter the numbers of rows ");
+   scanf("%d",&n);
+   for(i = 0; i < n; i++) 
```

Comment:
Having whitespace between logical blocks would enhance the readability of the code.